rest/data/project_settings.go

```diff
@@ -220,7 +220,7 @@ func SaveProjectSettingsForSection(ctx context.Context, projectId string, change
 				return nil, err
 			}
 		}
-		if err = validateFeaturesHaveAliases(mergedProjectRef, changes.Aliases); err != nil {
+		if err = validateFeaturesHaveAliases(mergedBeforeRef, mergedProjectRef, changes.Aliases); err != nil {
```

Comment:
Would it be crazy to rename mergedProjectRef to mergedSection so we remember in the future that this doesn't contain all of the sections?